bin/kafka-run-class.sh

```diff
@@ -20,6 +20,15 @@ then
   exit 1
 fi
 
+check_file() {
+  file=$1
+  if [ -z "$(echo "$file" | grep "test")" ] ; then
```

Comment:
There are other files included too. Basically anything that gets built into `*/build/libs/*` will show up be matched. 

We should be sure to exclude javadoc, scaladoc, sources, test and test-sources jars.